packages-content-model/roosterjs-content-model-editor/lib/publicApi/utils/paste.ts

```diff
@@ -120,14 +125,19 @@ function createBeforePasteEvent(
     };
 }
 
-function createFragmentFromClipboardData(
+function createBeforePasteEventAndFragment(
```

Comment:
Actually this fucntion does not only create event and fragment, but also trigger the event and let plugins handle it (same for the origianl code). So I think we need a better name.

Reply:
I renamed it to triggerPluginEventAndCreatePasteFragment and added a comment

Reply:
sounds good